src/extension.ts

```diff
@@ -62,10 +62,17 @@ export function activate(context: ExtensionContext) {
 
     // Register support for unicode input.
     (async () => {
-        const translations: any = await loadJsonFile.default(context.asAbsolutePath('translations.json'));
+        const translations : any = await loadJsonFile.default(context.asAbsolutePath('translations.json'));
+        const modes = [LEAN_MODE];
+        const input_markdown = workspace.getConfiguration('lean.input').get('markdown');
```

Comment:
Please install the tslint extension in vscode, and fix all errors.

Reply:
You can also do `npm run lint` to see all linting errors at once.

Reply:
done